pkg/api/scripts/remote-dev.js

```diff
@@ -3,6 +3,9 @@ const execSync = require('child_process').execSync;
 const helpers = require('../src/helpers');
 
 try {
+  if (process.env['NO_AUTH'] === 'true') {
```

Comment:
Let's use the environment variable `AUTH_REQUIRED` for consistency with the controller and other konveyor projects.